### PR TITLE
[IMP] l10n_ar_account_tax_settlement: nueva especificación txt iibb Misiones ret/perc

### PR DESCRIPTION
**Pull request by pablohmontenegro**
Ticket: 60295

### CODE REVIEW

**Comment by roboadhoc** [comment]
[Pull request status dashboard](https://mergebot.adhoc.ar/ingadhoc/odoo-argentina-ee/pull/191).

File: l10n_ar_account_tax_settlement/models/account_journal.py [hunk 1]
```diff
@@ -1279,15 +1279,23 @@ def misiones_files_values(self, move_lines):
                 # Fecha
                 content += fields.Date.from_string(payment.payment_date).strftime('%d-%m-%Y') + ','
 
-                # Constancia
-                content += payment.withholding_number[-8:] + ','
+                # Tipo de comprobante
+                #    Aquí vemos si se está pagando al menos una nota de crédito
+                #    si es así interpretamos que es corresponde a un CAR
+                matched_move_code_prefix = payment.payment_group_id.matched_move_line_ids.move_id.l10n_latam_document_type_id.mapped('doc_code_prefix')
```
Comment (pablohmontenegro) [inline l10n_ar_account_tax_settlement/models/account_journal.py:1285]: quizás en lugar de fijarme en el prefijo doc_code_prefix podría ver el move_type?

**Comment by zaoral** [comment]
@roboadhoc r+

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#221

**Comment by fw-bot-adhoc** [comment]
@pablohmontenegro @zaoral this pull request has forward-port PRs awaiting action (not merged or closed):
- ingadhoc/odoo-argentina-ee#221

**Pull request closed without merging**